### DISCUSSION TITLE
Allow direct connection IP Address

### DISCUSSION THREAD

**Discussion by cutpeter56** [rustdesk/rustdesk#1816] [Q&A]
Hi,
sorry for my english.

I deploy my own RustDesk Server on Debian. It's OK.
but I want to deploy RustDeskClient on my client PC running GPO. 
I rename the rustdesk.exe in rustdesk-host=192.X.X.X,key=azererzerterterty but I want to automatocally allow direct connection with the IP adress (with a port).

I test to write a newfile RustDesk2 and after the install, i copy this file to the folder %appdata%\RustDesk\config
```
rendezvous_server = '192.X.X.X'
nat_type = 2
serial = 3

[options]
enable-audio = 'N'
custom-rendezvous-server = '192.X.X.X'
direct-server = 'Y'
key = 'azeazeazeazea'
direct-access-port = '2118'
allow-remote-config-modification = 'Y'
```
But i doesn't work, the File was ecrased by the new file at the RustDesk opening.

I dont know how to do this.
Can you Help Me

**Comment by undee123** [comment]
Found a solution? That's exactly what I'm looking for now.

**Comment by cutpeter56** [comment]
nope... Nobody helps me

**Comment by Jorilx** [comment]
See [this issue](https://github.com/rustdesk/rustdesk/issues/769)

**Reply by pilatoroma** [reply]
# How to Configure RustDesk Direct IP Access Remotely on Windows (Without Physical Access)

## Problem

You need to enable Direct IP Access (`direct-server=Y`, port `21118`) on a remote Windows PC via RustDesk, but:
- Editing TOML files directly gets overwritten by the running service
- `--option` / `--config` CLI commands write to the **user profile** TOML, not the **service profile** TOML
- Running `Restart-Service` directly drops your remote connection before the command completes

## Solution (Tested & Working)

### Step 1 — Write options via CLI

```powershell
& "C:\Program Files\RustDesk\rustdesk.exe" --option direct-server Y
& "C:\Program Files\RustDesk\rustdesk.exe" --option direct-access-port 21118
